pagefaultgames/pokerogue
# [Enhancement] Add battle music for Koraidon & Miraidon

## What are the changes?

Adds one new track, the Depths of Area Zero Battle theme, for use as Koraidon and Miraidon's battle theme (as that is their battle theme in canon).

## Why am I doing these changes?



After a discussion with @damocleas , we decided that it would be fitting to give them this battle theme, instead of them just using the default Gen 5 legendary music.

## What did change?


Koraidon and Miraidon now have battle music. There is also one new entry in each BGM locale file.

### Screenshots/Videos



https://github.com/pagefaultgames/pokerogue/assets/59298170/c6dcfc3b-5b35-4e2e-b014-8294ca87970d

## How to test the changes?




Set the opponent override to either Koraidon or Miraidon. Make sure the music preference is set to Mixed.

## Checklist
